atk/src/attribute.rs

```diff
@@ -4,6 +4,17 @@ use glib::translate::*;
 use glib::GString;
 use std::fmt;
 
+/// AtkAttribute is a string name/value pair representing a generic
```

Comment:
You'll need to add the rustdoc-stripper annotation, otherwise it'll get removed.

Reply:
I didn't document it myself, it was always there :) I don't plan to fix gtk3, just the broken docs links

Reply:
The diff tells otherwise.

Reply:
probably caused by embed-docs, i will the changes there and regen.